release.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   "branches": [
-    "main"
+    "main",
+    { "name": "beta", "prerelease": true },
```

Comment:
Question:
Will every release be a beta release? How do we graduate the beta release without manually updating this file?

Reply:
Just like Kotlin CI, every release on the beta branch will be a beta release. To graduate the beta release, we need to merge the beta branch to the main branch.